package xerror

import "fmt"

func Error() {
	fmt.Println("xerror.error")
}
